### PR TITLE
QuantWiz-IQ tool for reporter based quantitation for proteins

### PR DESCRIPTION
**Pull request by pravs3683**
**New tool for Reporter based Isobaric Quantitation using iTRAQ and TMT tags from shotgun proteomics experiments**

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by dpryan79** [comment]
@bioconda-bot please update

**Comment by dpryan79** [comment]
I have no clue what's producing the "no such file or directory" error.

**Comment by dpryan79** [comment]
@bioconda-bot please fetch artifacts

**Comment by dpryan79** [comment]
I'm not entirely sure where the error is coming in. Presumably one could create a new env and install the package to test things manually.

**Comment by pravs3683** [comment]
Thank you Devon! I have modified the test section and it seems to be
linting successfully now.

File: recipes/quantwiz-iq/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+cp QuantWiz_IQ.pl $PREFIX/bin
+cp MODULES/*.pm $PREFIX/MODULES/
+
+cp -r Purity_correction $PREFIX/bin
+
+# Short wrapper script
+cat > $PREFIX/bin/QuantWiz_IQ <<EOF
+#!/bin/bash
+PERL5LIB=$module_dir QuantWiz_IQ.pl "\$@"
```
Comment (jj-umn) [inline recipes/quantwiz-iq/build.sh]: PERL5LIB=$module_dir exec $PREFIX/bin/QuantWiz_IQ.pl "\$@"

File: recipes/quantwiz-iq/build.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+mkdir -p $PREFIX/bin
+module_dir=$PREFIX/MODULES
+mkdir -p $module_dir
+
```
Comment (jj-umn) [inline recipes/quantwiz-iq/build.sh]: # Fix shebangs so perl found in environment
sed -i.bak 's|#!/usr/bin/perl|#!/usr/bin/env perl|' QuantWiz_IQ.pl

**Comment by jj-umn** [comment]
@pravs3683  I also commented on build.sh.   
One frequently needs to change the first line of scripts that indicates interpreter.  
Often developers use a absolute path:
**#!/usr/bin/perl** 
When it should be:
**#!/usr/bin/env perl**
to pick up the perl command at any location.

**Comment by pravs3683** [comment]
Thank you JJ!

**Comment by jj-umn** [comment]
@pravs3683 Can you squash and merge?   Or do you need me to do that?

**Comment by bgruening** [comment]
@jj-umn no need, this is done automatically.

**Comment by bgruening** [comment]
@BiocondaBot please merge

**Pull request merged**